fdm-app/app/routes/farm.$b_id_farm.$calendar.field.$b_id.cultivation.$b_lu._index.tsx

```diff
@@ -86,7 +90,11 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
         }
 
         // Get available cultivations for the farm
```

Comment:
fields also have an applicability for a given year?

Reply:
Waht do you mean with applicability?

Reply:
that fields are owned/used in a given year. should that also be checked / filtered?